interfaces/dbus/backend.go

```diff
@@ -55,6 +56,27 @@ func (b *Backend) Name() interfaces.SecuritySystem {
 	return "dbus"
 }
 
+// setupHostDBusConf will ensure that we have a dbus configuration
```

Comment:
In the case of other similar files we create we the trigger point is installation of either core or snapd. I wonder if we could behave similarly there, so that at least all that code is in one place.

Reply:
what's the relation of this code vs the one in wrappers/core18.go ?  the latter will be used only core, this one is for all systems, but wondering what are the various scenarios,  will we the files not be there early enough in some cases? also should we share code between the two paths?

Reply:
I'd written this code way back before the `wrappers/core18.go` code.  The intention was the same as the code installing user'd service activation files: ensure things work on a system with an old .deb installed snapd functions if it receives a new version via the `core` snap.

If we don't need that any more, then I guess we can remove this.  I wasn't sure the `wrappers/core18.go` code did anything on systems without the snapd snap.

Reply:
yes, wrappers/core18.go is for core >= 18 only, but maybe there's a way to have a helper in wrappers.go that is used in both cases